### ISSUE TITLE
Feature request - combine/break paper notes

### ISSUE THREAD

**Issue by ralphhogaboom** [thetealviper/Spigot_PaperMoney#2]
First off, THANK YOU FOR THIS PLUGIN! Holy smokes it's amazeballs. I'm using it as aphysical currency and it's been awesomesauce.

For those of us with a physical economy, having the ability to combine & break up notes would be huge. AKA I have 5 notes, each worth $1. I put them on a crafting bench, and it 'crafts' a single note worth $5.

If I put a single $10 note on there, it breaks it up into 1s and 5s. So it'd make 2 5s. 

Specifying your bill sizes like USD would make the most sense to me - 1s, 5s, 10s, 20s, 50s, 100s, 500s. 

This would be awesome. Combining & breaking a part bills to make change would be HUGE. 

Thank you for the awesome plugin!

**Comment by thetealviper** [comment]
I agree the idea of being able to split and combine is promising, however the implementation needs to be smooth. Some people's servers rely on commands, while others are more immersive and distribute the money in a way such that the player never touches a command. So I would need the integration of a split/merge feature to function in a way that pleases both parties. Crafting simply won't work because you can hack together a way to combine using crafting, but you can't possibly input 1 bigger bill and craft it into smaller bills through legit crafting because crafting recipes output one item. Sure I could probably hack together a way to make crafting LOOK legit, but it won't actually legitimately function the way you suggest as far as I'm aware. If you can think of a cleaner implementation that makes both types of servers I mentioned above happy then when I'm free I'll probably add it.

**Comment by thetealviper** [comment]
Added in [1.18.1.b.4]

**Comment by ralphhogaboom** [comment]
Thank you!
